tldw_Server_API/app/core/Evaluations/recipes/rag_answer_quality_execution.py

```
@@ -0,0 +1,1155 @@
+    db: Any,
+    service: Any = None,
+) -> dict[str, Any]:
+    baseline_ref = str(run_config.get("retrieval_baseline_ref") or "").strip()
+    if not baseline_ref:
+        raise ValueError("run_config.retrieval_baseline_ref is required for live_end_to_end runs.")
+
+    baseline_record = None
+    if service is not None:
+        try:
+            baseline_record = service.get_run(baseline_ref)
+        except Exception:
```

Ruff (0.15.7)

[warning] Do not catch blind exception: `Exception`

(BLE001)